Makefile

```diff
@@ -27,16 +29,23 @@ SHELL = /usr/bin/env bash -o pipefail
 # check if maven is installed
 ifeq (,$(shell which $(MAVEN)))
   $(error "No maven found in $(PATH). Please install maven")
+else
+  MVN_JAVA_VERSION=$(shell mvn --version | egrep -o "Java version: [[:digit:]]+\.[[:digit:]]+\.[[:digit:]]+"| head -n 1 | cut -c15-16)
```

Comment:
just a suggestion. I would use `sed` instead of `cut`. it is too rigit.
```
mvn --version | sed -rn 's/.*([[:digit:]]{2})\.[[:digit:]]+\.[[:digit:]]+(,|\s).*/\1/p'
```